app/layout.tsx

```diff
@@ -28,7 +28,7 @@ export default function RootLayout({
         <link rel="alternate icon" href="/SerotrackerLogo.png" />{" "}
         {/* Fallback for older browsers */}
       </head>
-      <body className={clsx(inter.className, "text-black no-scrollbar")}>
+      <body className={clsx(inter.className, "text-black no-scrollbar overflow-hidden")}>
```

Comment:
Commenting on random line.

This is a regression.

![image](https://github.com/serotracker/dashboard/assets/86806388/681510a2-9365-413e-af11-090d79b62552)

Reply:
regression?